Assets/Scripts/Game/EnemySenses.cs

```diff
@@ -624,7 +655,7 @@ public bool StealthCheck()
             return Dice100.FailedRoll(stealthChance);
         }
 
-        public bool BlockedByIllusionEffect()
+        public bool BlockedByIllusionEffectDefault()
```

Comment:
I'm a bit bothered by the change of a public API. After 1.0, this should generally be unacceptable. However, I doubt any mod is using this specifically when none of the other functionality is exposed.

Reply:
Oops, I didn't notice that was a public API.  I could revert the method name and rename the member if need be.

Reply:
I reverted the method name back to BlockedByIllusionsEffect.